d2l-floating-buttons.html

```diff
@@ -82,7 +82,12 @@
 			is: 'd2l-floating-buttons',
 
 			properties: {
-				_heightAtMost500px: Boolean
+				minHeight: {
+					reflectToAttribute: true,
+					type: String,
+					value: '500px'
+				},
+				_viewportIsMinHeight: Boolean
```

Comment:
Thinking about it, "_viewportIsAtLeastMinHeight" might be more accurate.  I can change it if it is much better.